gnomad_qc/v4/create_release/validate_and_export_vcf.py

```diff
@@ -451,6 +451,12 @@ def unfurl_nested_annotations(
         expr_dict["CMH_p_value"] = (
             ht.freq_comparison_stats.cochran_mantel_haenszel_test.p_value
         )
+        logger.info("Unfurling unionized stats...")
+        expr_dict["stat_union_p_value"] = ht.freq_comparison_stats.stat_union.p_value
+        expr_dict["stat_union_test"] = (
```

Comment:
```suggestion
        expr_dict["stat_union_test_name"] = (
```
Thoughts on this? I'm just going by KC explicitly adding `name` in slack: https://the-tgg.slack.com/archives/C06H5KM9W64/p1712931890171899

Reply:
Ah I missed this.